### ISSUE TITLE
Show maturity level

### ISSUE THREAD

**Issue by ProgrammerDan** [DevotedMC/Citadel#15]
Optionally show # of breaks to destroyed? Not sure how much is too much. To start with just a text label. -- "Immature, Very Little Maturation, Some Maturation, Decent Maturation, Largely Matured, Fully Matured" -- corresponding to 0%, 10%, 25%, 50%, 75%, 100%" 

Alternatively, time to maturity in minutes.

Alternatively, get rid of all these "intermediate" labels and construct a single "Current Strength: %" where we compute the _effective_ strength of a reinforcement against max and project it. 

How? Compute # of breaks needed to destroy the reinforcement. That divided by max health = current strength %.

So if we had a reinforcement that has a raw health of 1900 of 2000, and has a maturity of 10 minutes out of 100, w/ no decay, then we can compute that each hit causes 5 damage meaning 1900/5 breaks out of 2000 ideal breaks; (1900/5) / 2000 becomes our effective strength.

**Comment by Diet-Cola** [comment]
Are you looking to replace the % health with durability? I can probably do that

**Comment by ProgrammerDan** [comment]
You largely did this one by adding the percentage; iirc the idea was to pull together the 3/4 statuses now into one "percentage" of health left.

**Comment by Diet-Cola** [comment]
I also wanted to fix the part where a percentage can sometimes be like "97.00000001%".  

Would it be as simple as adding a .ToShort() on the end of getHealthText() [here?](https://github.com/DevotedMC/Citadel/blob/master/src/vg/civcraft/mc/citadel/reinforcement/PlayerReinforcement.java#L192)

**Comment by ProgrammerDan** [comment]
Nah, use String.format("%4.1f" ) or equivalent here: https://github.com/DevotedMC/Citadel/blob/master/src/vg/civcraft/mc/citadel/reinforcement/PlayerReinforcement.java#L136
